### PR TITLE
Update header button and misc styling.

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

- Further styling of initital claim page
- Restyled header claim button. Glowing effect stops when on the `/claim` page (opted for totally removing the claim button when _on_ the claim page, but for consistency purposes I think it's better to keep it at all times).

https://user-images.githubusercontent.com/31534717/149330431-6420cf8c-01bc-48bb-848c-5faaf499008a.mov

https://user-images.githubusercontent.com/31534717/149330450-b7495683-64ab-40d7-932c-d2128d9fb822.mov

Pending
- Further styling
- Mobile responsive solution for claim button in header.

### CODE REVIEW

**Review by W3stside — APPROVED** [review]
cool, let's merge

File: src/custom/components/Header/index.tsx [hunk 1]
```diff
@@ -265,19 +326,22 @@ export default function Header() {
...
                 {claimTxn && !claimTxn?.receipt ? (
                   <Dots>
                     <Trans>Claiming vCOW...</Trans>
                   </Dots>
                 ) : (
                   <>
                     <CowProtocolLogo />
-                    <Trans>vCOW</Trans>
+                    <b>
+                      <Trans>Claim vCOW</Trans>
```
Comment (anxolin) [inline src/custom/components/Header/index.tsx:338]: Why this change? 
My idea was to say "Claim vCOW", but afaik Anna changed it to vCOW only so we always show the button.

**Comment by anxolin** [comment]
The effects look great, by the way!

**Review by alfetopito — APPROVED** [review]
YES! 

Perfect except the `Claim` vCOW part

**Comment by biocom** [comment]
Consolidating waterfall. Review this PR at your convenience, post merge/consolidation.

**Comment by elena-zh** [comment]
Related issues are reported in #2148

**Review by anxolin — COMMENTED** [review]
APPROVE
